iOSNativeLoginTemplate/iOSNativeLoginTemplate/SceneDelegate.swift

```diff
@@ -129,3 +163,23 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
        postResetBlock()
    }
 }
+
+@MainActor class ReCaptchaClientObservable: ObservableObject {
+    
+    @Published var reCaptchaClient: RecaptchaClient? = nil
+    
+    init(reCaptchaSiteKey: String) {
+        Task(priority: .medium) {
+            await initializeReCaptchaClient(reCaptchaSiteKey: reCaptchaSiteKey)
+        }
+    }
+    
+    final func initializeReCaptchaClient(reCaptchaSiteKey: String) async {
+        do {
+
+            reCaptchaClient = try await Recaptcha.getClient(withSiteKey: reCaptchaSiteKey)
+        } catch let error {
+            SalesforceLogger.e(AppDelegate.self, message: "Cannot get reCAPTCHA client due to an error with description '\(error.localizedDescription).'.")
```

Comment:
should this be SceneDelegate or was there another logger issue?

Reply:
Good catch since the SceneDelegate is a class.  Updated!